requirements.txt

```diff
@@ -19,7 +19,7 @@ pydantic==1.8.2
 pyparsing==3.0.9
 pytest==7.1.2
 python-multipart==0.0.5
-PyYAML==5.4.1
+PyYAML==5.3.1
```

Comment:
@ysn100ysn it seems This issue was caused by an incompatibility of PyYAML<6.0.1 compiled under the version of Cython>=3.0, released months back. While downgrading works, upgrade to 6.0.1 is better as versions below 5.4 have some known issues. Please do this change so it can be merged. 
`PyYAML==6.0.1`